### PR TITLE
Replacing the Material 3 page with a more generic page

### PR DESCRIPTION
**Pull request by sfshaza2**
Staged:  https://sz-flutter-2.web.app/development/ui/material

Fixes: https://github.com/flutter/website/issues/8167

cc @guidezpl

### CODE REVIEW

**Review by domesticmouse — APPROVED** [review]

File: src/development/ui/material.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    become Flutter's default look and feel.
+  * Material 2 will eventually be removed according
+    to Flutter's [deprecation policy][].
+{{site.alert.end}}
+
+[Affected widgets]: {{site.api}}/flutter/material/ThemeData/useMaterial3.html#affected-widgets
+[deprecation policy]: {{site.url}}/resources/compatibility
+[GitHub umbrella issue]: {{site.github}}//flutter/flutter/issues/91605
+[demo]: https://flutter.github.io/samples/web/material_3_demo/#/
+[Material 3/Flutter GitHub project]: {{site.github}}/orgs/flutter/projects/19/views/22
```
Comment (parlough) [inline src/development/ui/material.md:42]: To fix the link error:

```suggestion
[Material 3 Flutter GitHub project]: {{site.github}}/orgs/flutter/projects/19/views/22
```

**Review by parlough — APPROVED** [review]

**Comment by domesticmouse** [comment]
Looks like the link checker is unhappy

```console
Checking for valid link references...
check_link_references: Invalid link references found!

development/ui/material/index.html:
[Material 3 Flutter GitHub project][]
make[1]: *** [Makefile:120: build-image-and-check-links] Error 1
```

File: src/development/ui/material.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+---
+title: Material Design for Flutter
+description: Learn about Material Design for Flutter.
+---
+
+Material is a design system created by Google to
+help teams build high-quality digital experiences
+for Android, iOS, Flutter, desktop, and the web.
+
+Material 3 is the latest generation of Material Design.
```
Comment (guidezpl) [inline src/development/ui/material.md:10]:
```suggestion
Material Design is an open-source design system built and supported by Google designers and developers.

The latest version, Material 3, enables personal, adaptive, and expressive experiences – from dynamic color and enhanced accessibility, to foundations for large screen layouts and design tokens.
```

File: src/development/ui/material.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    but you can follow progress on the
+    [Material 3 Flutter GitHub project][] and the
+    [GitHub umbrella issue][].
+  * For the latest list of Flutter's widgets that
+    are new and migrated to Material 3, see
+    [Affected widgets][].
+  * Until migration is complete, you can opt in to
+    Material 3 using the [`useMaterial3`][] property.
+    However, your UI might be inconsistent until
+    _all_ of Flutter _and_ your code is migrated.
```
Comment (guidezpl) [inline src/development/ui/material.md:27]:
```suggestion
  * You can opt in to
    Material 3 using the [`useMaterial3`][] flag.
    However, your UI might be inconsistent until
    _all_ of Flutter _and_ your code are migrated.
   * For the latest list of Flutter's widgets that
    are affected by the [`useMaterial3`][] flag, see
    [Affected widgets][].
```

File: src/development/ui/material.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  * Some Flutter widgets have similar behavior in M2 and
```
Comment (guidezpl) [inline src/development/ui/material.md:28]: "The vast majority of Flutter widgets..."

File: src/development/ui/material.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    M3, so those widget names are unchanged.
+    However, other widgets have substantially different
```
Comment (guidezpl) [inline src/development/ui/material.md:30]:
```suggestion
    However, a couple of widgets have substantially different
```

File: src/development/ui/material.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+    behavior in M3, so new widgets have been created.
+  * Once migration is complete, Material 3 will
+    become Flutter's default look and feel.
```
Comment (guidezpl) [inline src/development/ui/material.md:33]:
```suggestion
    become the Material library's default look and feel.
```

File: src/development/ui/material.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+  * Material 2 will eventually be removed according
```
Comment (guidezpl) [inline src/development/ui/material.md:34]:
```suggestion
  * Support for Material 2 will eventually be removed according
```

File: src/development/ui/material.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+{{site.alert.end}}
+
+[Affected widgets]: {{site.api}}/flutter/material/ThemeData/useMaterial3.html#affected-widgets
+[deprecation policy]: {{site.url}}/resources/compatibility
+[GitHub umbrella issue]: {{site.github}}//flutter/flutter/issues/91605
+[demo]: https://flutter.github.io/samples/web/material_3_demo/#/
+[Material 3/Flutter GitHub project]: {{site.github}}/orgs/flutter/projects/19/views/22
+[`useMaterial3`]: {{site.api}}/flutter/material/ThemeData/useMaterial3.html
+
+To play with the Material widgets, check out the
```
Comment (guidezpl) [inline src/development/ui/material.md:45]:
```suggestion
Explore the updated components, typography, color system and elevation support with the
```

File: src/development/ui/material.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+[interactive Material & Flutter showcase][demo]:
```
Comment (guidezpl) [inline src/development/ui/material.md:46]:
```suggestion
[interactive Material 3 demo][demo]:
```

File: src/development/ui/material.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+
+<iframe src="https://flutter.github.io/samples/web/material_3_demo/#/"
+        width="100%" height="500px" title="Material 3 Demo App"></iframe>
```
Comment (guidezpl) [inline src/development/ui/material.md:49]: is it possible to make this a bit taller, especially for big screens? Maybe 600 or some %?

File: src/development/ui/material.md [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+
+
+## More information
+{:.no_toc}
+
+To learn more about Material Design and Flutter,
+check out:    
+
+* [Flutter Material blog and announcements][]
```
Comment (guidezpl) [inline src/development/ui/material.md:58]:
```suggestion
* [Material.io developer documentation][]
```

**Review by guidezpl — CHANGES_REQUESTED** [review]
Looks great!

I have some minor suggestions for copy for clarity and to better align with Material's

File: src/development/ui/material.md (hunk 10 above)
Comment (sfshaza2) [reply src/development/ui/material.md]: Done

File: src/development/ui/material.md (hunk 1 above)
Comment (sfshaza2) [reply src/development/ui/material.md]: Thanks, @parlough! I had already fixed it in my local copy.

File: src/development/ui/material.md (hunk 4 above)
Comment (sfshaza2) [reply src/development/ui/material.md]: Done

**Review by guidezpl — APPROVED** [review]

**Pull request merged**